wiki/QFS-Client-Reference.md

```diff
@@ -283,3 +283,92 @@ _write-behind threshold_ by subsequent write calls or until user calls
 * _numBytes + pending >= write-behind threshold_: QFS client makes
 a copy of the source buffer only if _write-behind threshold_ is greater than
 zero. Write is performed in a blocking fashion.
+
+## Append Operations
+
+### RecordAppend
+
+RecordAppend can be used by a single writer to append to a replicated file. There is
+limited support for append to RS files.
+
+The file should be opened using the mode O_APPEND or O_WRONLY | O_APPEND.
+When RecordAppend is used to write a record, the entire record is guaranteed to be
+written to the same chunk. If the record will not fit in the current chunk, a new chunk
```

Comment:
Is it true that when you call "Close()" on the file, the subsequent "RecordAppend()" call always ends up in a new chunk? What's the case where the appended record goes to the end of the current chunk, but not to a new chunk?